use-timescale/tigerlake.md

```diff
@@ -0,0 +1,251 @@
+   1. Select `JSON` then replace the `Policy editor` code block with the following:
+   
+         ```json
+         {
+           "Version": "2012-10-17",
+           "Statement": [
+             {
+               "Sid": "BucketOps",
+               "Effect": "Allow",
+               "Action": [
+                 "s3tables:*"
+               ],
+               "Resource": "<S3TABLE_BUCKET_ARN>"
+             },
+             {
+               "Sid": "BucketTableOps",
+               "Effect": "Allow",
+               "Action": [
+                 "s3tables:*"
+               ],
+               "Resource": "<S3TABLE_BUCKET_ARN>/table/*"
+             }
+           ]
+         }
+         ```
+   1. Replace `<S3TABLE_BUCKET_ARN>` with the `Amazon Resource Name (ARN)` for the table bucket you just created.
+   1. Click `Next`, then give the inline policy a name and click `Create policy`.
+
+   Provide $COMPANY with the ARN of this role, the ARN of the S3 table bucket.
+  $COMPANY uses the outputs to spin up your $LAKE_LONG services, then let you know when the $SERVICE_SHORT is 
+  ready to use. Provisioning takes a couple of minutes, during this time the $SERVICE_SHORT is restarted.
+
+</Procedure>
+
+</Tab>
+
+</Tabs>
+
+## Stream data from your $SERVICE_LONG to your data lake
+
+To stream data from a $PG relational table, or a $HYPERTABLE in your $SERVICE_LONG to your data lake, run the following 
+statement:
+
+```sql
+ALTER TABLE <table_name> SET (
+  tigerlake.iceberg_sync = true | false,
+  tigerlake.iceberg_partitionby = '<partition_specification>'
+)
+```
+
+* `tigerlake.iceberg_sync`: `boolean`, set to `true` to start streaming, or `false` to stop the stream. A stream 
+  **cannot** resume after being stopped. 
+* `tigerlake.iceberg_partitionby`: optional property to define a partition specification in Iceberg. By default the 
+  partitioning specification of the $HYPERTABLE is used. Streamed $PG tables can have a partition specification 
+  for the Iceberg table, if intentially defined. Please refer to [partitioning](#partitioning) for more details.
+
+Only tables or $HYPERTABLEs with primary keys are supported, this includes composite primary keys as well. 
+A primary key is necessary for Iceberg to perform update or delete statements.
+
+When a stream is started, the full table is synchronized to Iceberg, this means that all prior records are imported first.
+The write throughput is ranging at approximately 40.000 records / second, for larger tables a full import can take some time.
+
+### Partitioning
+
+By default, the partition interval for an Iceberg table is the same as the one from a $HYPERTABLE.
+The sync of a Postgres table does not enable any partitioning in Iceberg, but can be set through the [API](#api) with `tigerlake.iceberg_partitionby`.
+
+The following partition intervals and specifications are supported, and the define behavior of [Iceberg partition specification][iceberg-partition-spec].
+
```

Comment:
Please remove the result-type column here.

Reply:
Done